cypress/integration/event-filter-page.js

```diff
@@ -1,31 +0,0 @@
-// describe('events page',() => {
```

Comment:
this file got deleted - we had a conversation about this but I think everything is fine. Looks like the tests that were covered here are covered elsewhere in other test files.